app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -129,31 +162,50 @@ export default {
         sku: null,
         price: '',
         currency: 'CLP',
-        quantity: '',
-        measure: '',
+        ingredient_measures_attributes: [], /* eslint-disable-line camelcase */
       },
+      showingMeasureModal: false,
+      usedUnits: [],
     };
   },
-
+  methods: {
+    AddMeasure() {
+      const lastItem = this.form.ingredient_measures_attributes[this.form.ingredient_measures_attributes.length - 1];
+      if (lastItem.name && lastItem.quantity) {
+        this.form.ingredient_measures_attributes.push({ name: undefined, quantity: undefined, id: undefined });
+      }
+    },
+    deleteUnit(unit) {
+      this.form.ingredient_measures_attributes = this.form /* eslint-disable-line camelcase */
+      .ingredient_measures_attributes.filter((originalUnit) => originalUnit !== unit);
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 8 spaces but found 6  indent